vertical-pod-autoscaler/hack/run-e2e-tests.sh

```diff
@@ -45,13 +45,7 @@ SUITE=$1
 case ${SUITE} in
   recommender|updater|admission-controller|actuation|full-vpa)
     export KUBECONFIG=$HOME/.kube/config
-    go test ${SCRIPT_ROOT}/e2e/v1beta1/*go -v -test.timeout=60m  --args --ginkgo.v=true --ginkgo.focus="\[VPA\] \[${SUITE}\]" --report-dir=/workspace/_artifacts --disable-log-dump
-    FIRST_RESULT=$?
     go test ${SCRIPT_ROOT}/e2e/v1beta2/*go -v -test.timeout=60m  --args --ginkgo.v=true --ginkgo.focus="\[VPA\] \[${SUITE}\]" --report-dir=/workspace/_artifacts --disable-log-dump
```

Comment:
Since we have one script again we may restore "set -o errexit"

Reply:
Done